extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/handler/HomematicThingHandler.java

```diff
@@ -339,9 +339,8 @@ private void updateChannelState(ChannelUID channelUID)
      */
     protected void updateDatapointState(HmDatapoint dp) {
         try {
-            if (HomematicTypeGeneratorImpl.isStatusDatapoint(dp)) {
```

Comment:
Looks like `HomematicTypeGeneratorImpl.isStatusDatapoint(HmDatapoint)` is not required any longer. You may remove it.

Reply:
You're right, I'll remove the method.